ofrak_core/ofrak/service/dependency_handler.py

```diff
@@ -65,6 +66,19 @@ async def handle_post_patch_dependencies(self, patch_results: List[DataPatchesRe
             patch_result.data_id for patch_result in patch_results
         )
 
+        all_data_ids = [patch_result.data_id for patch_result in patch_results]
+        models_by_data_id: Dict[bytes, DataModel] = {
+            data_id: model
+            for data_id, model in zip(
+                all_data_ids, await self._data_service.get_by_ids(all_data_ids)
+            )
+        }
+        # First go through and update all models' Data
```

Comment:
```suggestion
        # Go through and update all models' Data
```

"First" sort of implies that there will be a "next" in a subsequent comment...